### PR TITLE
Added Total Doctors Count card on Facility home page

### LINKED ISSUE #4731: Total Doctors count card on facility page
**Describe the bug**
Can we add a Total Doctors count card on the Facility page just like the Total Bed capacity card?
https://care.coronasafe.in/facility/711bd31a-feeb-4a9e-9893-b317579bfff8

**Screenshots**
image

Bed capacity
image

Sample view
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by Pranshu1902**
Fixes #4731 
Added total doctors count card
http://localhost:4000/facility/711bd31a-feeb-4a9e-9893-b317579bfff8
image

### CODE REVIEW

**Comment by nihal467** [comment]
@Pranshu1902 
image

when I update the count of an existing doctor capacity, the total doctor count card is not showing the updated count, unless I manually refresh the whole page.

**Comment by Pranshu1902** [comment]
@nihal467 bug resolved!

**Comment by nihal467** [comment]
LGTM

File: src/Components/Facility/FacilityHome.tsx [hunk 1]
```diff
@@ -96,6 +98,15 @@ export const FacilityHome = (props: any) => {
             }
             if (doctorRes && doctorRes.data) {
               setDoctorData(doctorRes.data.results);
+              // calculating total doctors count
+              let totalCount = 0;
+              console.log(doctorRes.data.results[0]);
```
Comment (khavinshankar) [inline src/Components/Facility/FacilityHome.tsx]:
```suggestion
```

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**